src/content/app/entity-viewer/gene-view/GeneView.tsx

```diff
@@ -186,9 +184,7 @@ const GeneViewWithData = (props: GeneViewWithDataProps) => {
           {props.gene.transcripts.length > 5 && (
             <div className={styles.filterLabelWrapper}>
               <ShowHide
-                classNames={{
-                  label: filterLabelClass
-                }}
+                className={isFilterPanelOpen ? styles.showHide : undefined}
```

Comment:
Is it better to write undefined or ''? In my PR I used the latter

Reply:
`className={undefined}` will result in no `class` attribute on the DOM element:

```
<span>Hello</span>
```

whereas `className=""` will most likely result in an empty `class` attribute on the DOM element (unless React has changed its behaviour):

```
<span class>Hello</span>
```

I think the first result is cleaner.